packages/ssx-react/src/next/frontend.ts

```diff
@@ -0,0 +1,31 @@
+import { getCsrfToken, signIn, signOut } from 'next-auth/react';
+import { SSXClientSession } from '@spruceid/ssx';
+
+export const SSXNextAuthRouteConfig = () => {
+  const nonce = {
+    customOperation: async () => await getCsrfToken(),
+  };
+  const login = {
+    customOperation: async (session: SSXClientSession) => {
+      const callbackUrl = '/protected';
```

Comment:
Do we want to accept params in login that can be passed here? How do we let the user pass their own `callbackUrl` in Approach A? This function is called in `ssx.login()` (when configured for NextAuth)

Reply:
Yes, I think so. We should add something like `signInParams` in the SSXNextAuthRouteConfig.